usage.md

```diff
@@ -285,4 +285,18 @@ returns `true` if required.
 
 ```js
 const isPermissionRequired = await storage.checkPermission() -> boolean
+```
+
+### Add existing file to App
```

Comment:
What is the purpose of adding a file that already exists in the Arcana Store into a dApp?